src/ui/src/app/chart-details/chart-details-info/chart-details-info.component.ts

```diff
@@ -26,6 +26,21 @@ export class ChartDetailsInfoComponent implements OnInit {
     return this.chart.attributes.sources || [];
   }
 
+  get sourceUrl(): string {
+    var chartSource = this.chart.attributes.repo.sourceURL;
+    if (!chartSource) return
+
+    // Used to handle possible trailing URLs
+    var urljoin = require('url-join');
+    return urljoin(chartSource, this.chart.attributes.name);
+  }
+
+  get sourceName(): string {
+    var parser = document.createElement('a');
+    parser.href = this.chart.attributes.repo.sourceURL;
+    return parser.protocol + "//" + parser.hostname;
```

Comment:
OK, removed. I added it since the rest of the links in related appear with the protocol. But since now that I have changed the order (being on top of maintainers), removing the protocol makes sense.